src/meta/proto-conv/src/share_meta_v1_from_to_protobuf_impl.rs

```diff
@@ -59,6 +59,9 @@ impl FromToProto for mt::ShareGrantObject {
             mt::ShareGrantObject::Table(table_id) => {
                 Some(pb::share_grant_object::Object::TableId(*table_id))
             }
+            mt::ShareGrantObject::View(_table_id) => {
+                unreachable!()
```

Comment:
Why panic?

Reply:
ShareMetaV1 is deprecated, so in view case is unreachable